Makefile

```diff
@@ -12,17 +12,19 @@ PKG 		:= github.com/envoyproxy/go-control-plane
 build:
 	@go build ./pkg/... ./envoy/...
 
+# TODO(mattklein123): See the note in TestLinearConcurrentSetWatch() for why we set -parallel here
+# This should be removed.
 .PHONY: test
 test:
-	@go test ./pkg/...
+	@go test -race -v -timeout 30s -parallel 100 ./pkg/...
 
 .PHONY: cover
 cover:
 	@build/coverage.sh
 
-.PHONY: format
-format:
-	@goimports -local $(PKG) -w -l pkg
+.PHONY: check_format
+check_format:
+	@gofmt -l $(shell go list -f '{{.Dir}}' ./...) | tee /dev/stderr | read && echo "Files failed gofmt" && exit 1 || true
```

Comment:
Should we still be using goimports? iiuc it does what go fmt does but more

Reply:
I see let me check that. It didn't seem like goimports worked in the new container so I just made this what ratelimit was doing.